molecule/default/tests/test_app.yml

```diff
@@ -0,0 +1,86 @@
+---
+## TODO: Remember to adapt goss tests to your convenience
+
+#service:
```

Comment:
shouldn't these be removed if they are not going to be used?

Reply:
True, that should be removed

Reply:
Done